lib/public/js/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
 console.log('Hello, pals!');
+
+const setActiveNavItems = () => {
+
+    const list = document.getElementsByClassName('nav__item');
+    const length = list.length;
+    const currentPage = window.location.href;
+
+    let iterator;
+    for (iterator = 0; iterator < length; ++iterator) {
```

Comment:
It's typical to write `let iterator = 0` in here, so that the variable name `iterator` is scoped only to the `for` block.